packages/lending/src/protocols/compound-v3/lending-protocol.ts

```diff
@@ -326,12 +339,15 @@ export class LendingProtocol extends Protocol {
       });
     }
 
+    const borrowGrossApy = lstApy === '0' ? borrowAPR : BigNumberJS(borrowAPR).minus(lstApy).toString();
```

Comment:
Do you think adding the condition check `lstApy === '0'` into the utility is a good way to keep the code clean?

Reply:
added `calcSupplyGrossApy` and `calcBorrowGrossApy` to make the code clean